clients/vscode-hlasmplugin/.gitignore

```diff
@@ -1,11 +1,13 @@
-node_modules/
-dist/
-dist_test/
-lib/
+/.nyc_output/
+/.vscode-test/
+/.vscode-test-web/
+/bin/
+/coverage/
+/dist/
+/dist_test/
+/lib/
+/lib_syntaxes/
+/node_modules/
+/syntaxes/
```

Comment:
There are 2 tracked files in the syntaxes folder, right? Now that it became a sort of a build folder and places in the .gitignore file, I think the PR should delete both files from the repo.

Reply:
👍  Hmmm, I thought I removed both of them but apparently not

Reply:
Turned out that git considers the second file moved instead of removed:

![image](https://github.com/BroadcomMFD-DevOps/che-che4z-lsp-for-hlasm/assets/99467904/003ee8e3-3783-44ad-b1da-f154c0a88f6c)

So this is what happened to the two files:
- clients/vscode-hlasmplugin/syntaxes/hlasm.tmLanguage.json - moved
- clients/vscode-hlasmplugin/syntaxes/hlasmListing.tmLanguage.json - removed


=> The contents of the `syntax` directory is indeed no longer tracked as can be verified also in codespaces for this PR (not sure if the link works for you): https://potential-system-9gjp6jprgrrfp6w5.github.dev/